embuilder.py

```diff
@@ -33,7 +33,6 @@
         libcxx_noexcept
         libcxxabi
         gl
-        struct_info
```

Comment:
I mean, practically no regular user would run those tests. e.g. the bots don't even.